src/v0/destinations/braze/util.js

```diff
@@ -123,11 +125,12 @@ const BrazeDedupUtility = {
   async doApiLookup(identfierChunks, destination) {
     return Promise.all(
       identfierChunks.map(async (ids) => {
```

Comment:
how are we ensuring that we respect the braze API limits?

Reply:
We are not having any active checks it more passive. If the braze API throttles we move ahead without updating `store` and proceed with duplicates